from pylint_exception_var_name_plugin.exception_var_name import register
